### PR TITLE
[ADO]policy project and enable attestation changes

### PR DESCRIPTION
**Pull request by arvindsingh-cloud**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1 [hunk 1]
```diff
@@ -99,7 +97,12 @@ class ADOSVTBase: SVTBase {
 		$controlState = @();
 		$controlStateValue = @();
 		try {
-			$resourceStates = $this.GetResourceState($false)			
+			$resourceStates = $this.GetResourceState($false)
+			if (!$this.AzskSettings) {
+				$this.AzskSettings = [ConfigurationManager]::GetLocalAzSKSettings();
+			}	
+			$enableOrgControlAttestation = $this.AzskSettings.EnableOrgControlAttestation
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1:104]: shouldn't this also be inside the above if condition?

File: src/AzSK.ADO/Framework/Managers/ControlStateExtension.ps1 [hunk 2]
```diff
@@ -435,17 +435,24 @@ class ControlStateExtension
...
 			if ([string]::IsNullOrEmpty($projectName))
 			{
 				$projectName = $this.InvocationContext.BoundParameters["PolicyProject"]
 				if ([string]::IsNullOrEmpty($projectName))
 				{
-					if ($this.PrintParamPolicyProjErr -eq $true)
+					if (!$this.AzskSettings) {
+						$this.AzskSettings = [ConfigurationManager]::GetLocalAzSKSettings();
+					}
+					$projectName = $this.AzskSettings.PolicyProject
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Managers/ControlStateExtension.ps1:447]: shouldn't this be inside the above if condition?
shouldn;t it be made static?

File: src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1 (hunk 1 above)
Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1:104]: no above condition means load $this.AzskSettings only if not already loaded

File: src/AzSK.ADO/Framework/Managers/ControlStateExtension.ps1 (hunk 2 above)
Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/Managers/ControlStateExtension.ps1:447]: not inside

**Pull request merged**